package.json

```diff
@@ -3,8 +3,23 @@
   "version": "8.3.0",
   "description": "JSON Web Token implementation (symmetric and asymmetric)",
   "main": "index.js",
+  "nyc": {
+    "check-coverage": true,
+    "lines": 95,
+    "statements": 95,
+    "functions": 100,
+    "branches": 95,
+    "exclude": [
+      "./test/**"
+    ],
+    "reporter": [
+      "json",
+      "lcov",
+      "text-summary"
```

Comment:
does this mean no HTML reporter?

Reply:
Preferably it should be 100% coverage, but as we are missing some tests on exception handling and @MitMaro is in the process of refactoring the tests, i was being liberate ;-)

Lcov includes a html reporter :-) 

<img width="238" alt="screen shot 2018-06-25 at 20 54 35" src="https://user-images.githubusercontent.com/5888337/41870048-2b666eaa-78bb-11e8-9a62-6ee26129d7a4.png">